File: Contact.DAL/ContactType.cs
using System;
using System.Collections.Generic;

namespace Contact.DAL
{
    public partial class ContactType
    {
        public ContactType()
        {
            CustomerContacts = new HashSet<CustomerContact>();
        }

        public int Id { get; set; }
        public string TypeName { get; set; } = null!;
        public string ValidationExpression { get; set; } = null!;

        public virtual ICollection<CustomerContact> CustomerContacts { get; set; }
    }
}

File: contact-app/src/models/ContactType.ts
export class ContactType {
    id!:number;
    typeName!:string;
    validationExpression!:string;

    constructor(model:any = null) {
        this.id = model?.id;
        this.typeName = model?.typeName;
        this.validationExpression = model?.validationExpression;
    }
}